docs/elasticsearch-docs.asciidoc

```diff
@@ -1,21 +1,38 @@
 [[elasticsearch-docs]]
 == Elasticsearch
 
-The documents produced by APM Server have a specific structure.
-There are examples of such documents as well as documentation 
-
+The APM Server produces documents with a specific structure.
+Based on the defined structure a template mapping can be used for Index creation in Elasticsearch.
+See examples below for different document types. 
 
 === Example docs
 
-The following are example documents sent to Elasticsearch from APM Server.
+In the following you find example documents sent to Elasticsearch from APM Server.
+
+==== Transaction document
+
+[source,json]
+----
+include::./data/elasticsearch/transaction.json[]
+----
+
+==== Trace document
 
-link:./data/elasticsearch/transaction.json[Transaction Document]
+[source,json]
+----
+include::./data/elasticsearch/trace.json[]
+----
 
-link:./data/elasticsearch/trace.json[Trace Document]
+==== Error document
 
-link:./data/elasticsearch/error.json[Error Document]
+[source,json]
+----
+include::./data/elasticsearch/error.json[]
+----
 
 
+===  Indexed Fields
 
-include::./fields.asciidoc[See indexed fields in Elasticsearch]
+See which fields are indexed in Elasticsearch:
```

Comment:
Instead of adding a section, it would be better to add a cross reference that points to the section about exported fields. When you look at the output, you get a topic that contains hardly any content.

![image](https://user-images.githubusercontent.com/14206422/30559626-1ba3e1c4-9c6a-11e7-8c96-8308add9e3d8.png)

I don't think there's any real value in this section. Maybe at the top of the file (before you provide examples), you should have a sentence that says:  "To see which fields are indexed in Elasticsearch, see Exported Fields" (make Exported Fields an active link).